cypress/integration/finance/funds/add-allocation-to-budget.spec.js

```diff
@@ -0,0 +1,79 @@
+import permissions from '../../../support/dictionary/permissions';
+import testType from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import Users from '../../../support/fragments/users/users';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import InteractorsTools from '../../../support/utils/interactorsTools';
+
+describe('ui-finance: Add budget to fund', () => {
+  const defaultfund = { ...Funds.defaultUiFund };
+  const defaultFiscalYear = { ...FiscalYears.defaultUiFiscalYear };
+  const defaultLedger = { ...Ledgers.defaultUiLedger };
+  const allocatedQuantity = '50';
+  let user;
+
+  before(() => {
+    cy.getAdminToken();
+    FiscalYears.createViaApi(defaultFiscalYear)
+      .then(response => {
+        defaultFiscalYear.id = response.id;
+        defaultLedger.fiscalYearOneId = defaultFiscalYear.id;
+
+        Ledgers.createViaApi(defaultLedger)
+          .then(ledgerResponse => {
+            defaultLedger.id = ledgerResponse.id;
+            defaultfund.ledgerId = defaultLedger.id;
+
+            Funds.createViaApi(defaultfund)
+              .then(fundResponse => {
+                defaultfund.id = fundResponse.fund.id;
+
+                cy.loginAsAdmin({ path:TopMenu.fundPath, waiter: Funds.waitLoading });
+                FinanceHelp.searchByName(defaultfund.name);
+                FinanceHelp.selectFromResultsList();
+                Funds.addBudget(allocatedQuantity);
+              });
+          });
+      });
+    cy.createTempUser([
+      permissions.uiFinanceCreateAllocations.gui,
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        cy.login(userProperties.username, userProperties.password);
```

Comment:
use path parameter here like in loginAsAdmin function

Reply:
DOne